NFR.md

```diff
@@ -145,7 +146,7 @@ English throughout.
 5.1.4 Source Code MUST be marked with licensing information according to
 MIT license rules. Copyright MUST be clearly defined.
 
-5.1.5 New X-Road components MUST be created using Java 11.
+5.1.5 New X-Road components MUST be created using the same Java version with the X-Road core.

```

Comment:
Perhaps using the term `X-Road core` is a bit unclear for readers outside the project in this context, since there isn't one specific module we can direct the users at for guidance. Maybe it would be better to say it like this:
```suggestion
5.1.5 New X-Road components MUST be created using the same Java version as the existing components.
```